src/dev/build/lib/fs.js

```diff
@@ -39,6 +39,10 @@ const readFileAsync = promisify(fs.readFile);
 const readdirAsync = promisify(fs.readdir);
 const utimesAsync = promisify(fs.utimes);
 
+export function isFileAccessible(path) {
```

Comment:
All the other fs module functions validate that the path is absolute.

Reply:
👍